firmware/application/transmitter_model.hpp

```diff
@@ -26,77 +27,68 @@
 #include <cstdint>
 #include <cstddef>
 
-#include "receiver_model.hpp"
 #include "app_settings.hpp"
+#include "max2837.hpp"
 #include "message.hpp"
+#include "receiver_model.hpp"
 #include "rf_path.hpp"
-#include "max2837.hpp"
-#include "volume.hpp"
 #include "signal.hpp"
 
 class TransmitterModel {
    public:
+    struct settings_t {
```

Comment:
I pulled the actual settings into structs to make them easier to pass around (for overrides) and initialize -- now there's only one place where default initialization happens.